controllers/release/release_adapter.go

```diff
@@ -229,25 +229,29 @@ func (a *Adapter) EnsureSnapshotEnvironmentBindingIsCreated() (results.Operation
 // EnsureTargetContextIsSet is an operation that will ensure that the targetContext of the adapter is not nil. It will
 // set it to a context from the workspace in the ReleasePlan if one is defined or to its own context if not.
 func (a *Adapter) EnsureTargetContextIsSet() (results.OperationResult, error) {
```

Comment:
Where is the release status being set in a non KCP env?

Reply:
Is it required? I thought the Status would be important only to set the context workspace. I understood that the context is the same in non-KCP environments.

Reply:
Status should be updated for both as we want to be consistent.

Reply:
Fixed.